.gitignore

```diff
@@ -62,3 +62,4 @@ props-table
 **/__snapshots__/diff
 **/__snapshots__/error
 generated-themes
+package-lock.json
```

Comment:
Why is this being added to the `.gitignore`? It will not have any affect since `package-lock.json` is already committed and tracked.